nil/services/rpc/jsonrpc/txpool_api_test.go

```diff
@@ -47,7 +47,7 @@ func (suite *SuiteTxnPoolApi) SetupSuite() {
 	defer database.Close()
 
 	suite.api = rawapi.NodeApiBuilder(database, nil).
-		WithLocalShardApiRo(types.MainShardId, suite.pool).
+		WithLocalShardApiRw(types.MainShardId, suite.pool).
```

Comment:
That looks like a fixing a typo from previous patch

Reply:
Actually, no. Moving the methods that work with txpool to the rw API is a separate sub-task. First I kept the status quo and then made these necessary changes.